docs/content/concepts/assets/graph-backed-assets.mdx

```diff
@@ -51,79 +51,60 @@ def store_files(files):
     return files.to_sql(name="slack_files", con=create_db_connection())
 
 
-@graph
-def store_slack_files_in_sql():
+@graph_asset
+def slack_files_table():
     return store_files(fetch_files_from_slack())
-
-
-graph_asset = AssetsDefinition.from_graph(store_slack_files_in_sql)
 ```
 
 ### Defining basic dependencies for graph-backed assets
 
-The `from_graph` attribute on the `AssetsDefinition` object infers upstream and downstream asset dependencies from the graph definition provided. In the most simple case when the graph returns a singular output, Dagster infers the name of the graph to be the outputted asset key.
+Similar to with basic op-backed assets, Dagster infers the upstream assets from the names of the arguments to the decorated function.
```

Comment:
> basic op-backed assets

This strikes me as potentially confusing, as we don't typically talk about assets in this way. Aren't all assets essentially backed by ops? Wouldn't `basic software-defined assets` be the same thing?

Reply:
👍 👍